sentence.go

```diff
@@ -87,6 +93,9 @@ func parseSentence(raw string) (BaseSentence, error) {
 
 // parsePrefix takes the first field and splits it into a talker id and data type.
 func parsePrefix(s string) (string, string) {
+	if s == TypeGSensord {
```

Comment:
Since we are adding a new, empty, prefix for Gsensord, we should add a test case for this.